app/main.py

```diff
@@ -1,3 +1,12 @@
 def cache(func):
-    # Write your code here
-    pass
+    functions = dict()
```

Comment:
Could you change the name of this variable? It seems that some functions are stored in it